From 28f7def4bf579cf49b847301fcbf761b2a2ea8c8 Mon Sep 17 00:00:00 2001
From: Matej Habrnal <mhabrnal@redhat.com>
Date: Fri, 4 Sep 2015 10:06:16 +0200
Subject: [PATCH] doc: add example into the abrt-auto-reporting man page

Add example how to turn on the authentication using rhsm consumer certificate.

Related to rhbz#1223805

Signed-off-by: Matej Habrnal <mhabrnal@redhat.com>
---
 doc/abrt-auto-reporting-authenticated.txt | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/doc/abrt-auto-reporting-authenticated.txt b/doc/abrt-auto-reporting-authenticated.txt
index 2a27945..de3302d 100644
--- a/doc/abrt-auto-reporting-authenticated.txt
+++ b/doc/abrt-auto-reporting-authenticated.txt
@@ -97,6 +97,14 @@ OPTIONS
    Also turns the HTTP Authentication off, because these methods cannot
    be used together.
 
+EXAMPLES
+--------
+Turn on the authentication using rhsm consumer certificate:
+
+------------
+# abrt-auto-reporting enabled --certificate rhsm
+------------
+
 SEE ALSO
 --------
 abrt.conf(5), ureport.conf(5), rhtsupport.conf(5)
-- 
2.4.3

